spec/factories/recipes.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: recipes
+#
+#  id          :bigint           not null, primary key
+#  name        :string           not null
+#  description :text             not null
+#  ingredients :string           not null
+#  user_id     :bigint           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_recipes_on_user_id  (user_id)
+#
+FactoryBot.define do
+  factory :recipe do
+    name { 'MyString' }
```

Comment:
What if you use some fake data to ensure randomness in testing? Check https://github.com/faker-ruby/faker

Reply:
Nice suggestions, i never imagined finding fakers even for food names 😆